exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -143,9 +143,9 @@ func (prwe *PrwExporter) PushMetrics(ctx context.Context, md pdata.Metrics) (int
 			}
 		}
 
-		if err := prwe.export(ctx, tsMap); err != nil {
+		if exportErrors := prwe.export(ctx, tsMap); exportErrors != nil {
```

Comment:
so, `prwe.export` returns slice of errors. What if the slice is empty? Does that still indicate error condition? 

In other words, is checking nil like `exportsErrors != nil` still correct given that `exporteErrors` is a slice? 

Typically, to avoid ambiguity like this, I would have some kind of aggregate data type that encompassed multiple error such that I can just still do a `nil` check.  Does GoLang have some kind of aggregated error type that allows you to append multiple errors?

Reply:
I should be using `len(exportErrors) == 0` here as it's an array of errors. I believe this method of aggregating errors into a list (and combining them together later) is already used in the PRWExporter code. I'll add this in.